test/test-install.sh

```diff
@@ -23,3 +22,36 @@ if ! [ -z "$(find "${PREFIX}" -type f)" ]; then
         find "${PREFIX}" -type f
         exit 1
 fi
+
```

Comment:
This patch doesn't look right.

Reply:
Maybe it is a little bit over the top but it tests "every" make scenario for that change

Reply:
Ahh I see, do you mean the single dunstify test with the systemd options and stuff?

Reply:
Well, I do not read code in a review, which is obviously copied multiple times. How am I supposed to find the bits, which are different? How am I supposed understand the differences in the future? Would a comment help? (spoiler: no) ...

Keep your IT DRY. [Here's a perfect article to summarize it.](https://michael.lustfield.net/pages/keeping-it-dry)

Reply:
I'm not sure I get what's the point of testing dunstify installation separately, it doesn't have any special cases to test for?

The mistake here was probably not modifying `config.mk` to always include libnotify in the dependencies.

Reply:
> How am I supposed to find the bits, which are different? How am I supposed understand the differences in the future? Would a comment help? (spoiler: no) ...
> 

You are completely right, I didn't think about that. Thanks for the article!